dcompiler.dd

```diff
@@ -864,6 +865,23 @@ make -f posix.mak DMD=~/$(DMDDIR)/$(OS)/bin/dmd
 )
 
 
+$(H2 $(LEGACY_LNAME2 tips_tricks, tips-tricks, Tips and Tricks))
```

Comment:
You don't need to start new things with legacy support ;-)
Just `$(H2 $(LNAME2 tips-tricks, Tips and Tricks))` should be fine